google_guest_agent/command/command.go

```diff
@@ -0,0 +1,140 @@
+//  Copyright 2023 Google Inc. All Rights Reserved.
```

Comment:
what's the difference between `command.go` and `command_monitor.go`?

Reply:
`command.go` contains the API for handlers (registration, deregistration, CmdNotFound status codes, etc). They are all the same package of course, I just split it in such a way that someone reading through the API to implement a handler wouldn't be seeing implementation details.

Reply:
Ok, that should be fine.